README.adoc

```diff
@@ -21,7 +21,24 @@ Check back for updates, we plan to have something available "soon".
 [[documentation]]
 == Documentation
 
-- link:./developing.adoc[Developing CodeReady Containers]
+=== Using CodeReady Containers
+
+You can find the raw documentation files link:./docs/source[here]
+
+However, to build a nicely formatted html document do this:
+
+```bash
+$ cd $HOME
```

Comment:
I don't believe this is a necessary step.

Reply:
ack, it's not, I added that only so the path I mentioned later could be absolote :)